languages/en.ini

```diff
@@ -172,6 +172,8 @@ Catalog Results = "Catalog Results"
 catalog_login_desc = "Enter your library catalog credentials."
 CD = "CD"
 Change Password = "Change Password"
+channel_expand_icon = "Show more channels like this one"
+channel_search_icon = "Show channel as search results"
```

Comment:
I changed the icons, but not the language yet. I'm leaning more towards "Show as list" and something like "Focus on this channel". The latter new icon is a search-glass with a plus-sign in it, like a zoom-in option. I think it's a better, but not perfect metaphor.